lib/ufe/UsdContextOps.h

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright 2020 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#pragma once
+
+#include "../base/api.h"
+
+#include "UsdSceneItem.h"
+
+#include <ufe/path.h>
+#include <ufe/contextOps.h>
+
+#include <pxr/usd/usd/prim.h>
+
+PXR_NAMESPACE_USING_DIRECTIVE
+
+MAYAUSD_NS_DEF {
+namespace ufe {
+
+//! \brief Interface for scene item operations.
+class MAYAUSD_CORE_PUBLIC UsdContextOps : public Ufe::ContextOps
+{
+public:
+	typedef std::shared_ptr<UsdContextOps> Ptr;
+
+	UsdContextOps();
+	~UsdContextOps() override;
+
+	// Delete the copy/move constructors assignment operators.
```

Comment:
It could be a personal preference, but expectations about what's allowed to be done with the class should be provided in the class description.

Reply:
Agreed. Useful comments explain _why_ you are doing something, rather than _what_ you are doing (the _what_ can be gleaned from the code)